public/css/main.css

```diff
@@ -1335,3 +1333,93 @@ kbd {
     top: -85px;
     visibility: hidden;
 }
+
+@media only screen and (max-width: 990px) {
```

Comment:
Could you add a link to some documentation you used to set up the CSS for the hamburger menu here?

Reply:
I didn't really use documentation for the CSS of the hamburger menu. I just used Bootstrap 3.3.5's default navbar and override some CSS. Would just linking the bootstrap 3.3.5 navbar documntation be fine?

Reply:
yep!